Scripts/fn_respiratorEffects.sqf

```diff
@@ -0,0 +1,68 @@
+/*
+ * Author: Alganthe, TyroneMF
+ * Specified masks protect from a contamination zone while providing HUD/Sound effects.
+ * Requires a marker covering an area named "Contamination" for damage to take effect.
+ *
+ * Call from initPlayerLocal.sqf
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_player] call FUNC(respiratorEffects);
+ */
+#include "..\script_component.hpp"
+
+params ["_player"];
+
+GVAR(maskCounter) =  CBA_missionTime;
+GVAR(lastSoundRan) = CBA_missionTime;
+GVAR(oldGlasses) = "";
+[{
+    private _masks = [
```

Comment:
This has no reason to be inside the PFH. Make it a `#define` above.
Also lower-case them all (and add a comment about it) - look at below comment.